sa/model.go

```diff
@@ -25,6 +30,23 @@ type dbSelector interface {
 	Select(interface{}, string, ...interface{}) ([]interface{}, error)
 }
 
+// a `dbInserter` is anything that provides an `Insert` function
+type dbInserter interface {
+	Insert(list ...interface{}) error
+}
+
+// A `dbExecer` is anythin that provides an `Exec` function
+type dbExecer interface {
+	Exec(string, ...interface{}) (sql.Result, error)
+}
+
+// selectExecer offers a subset of gorp.SqlExecutor's methods: Select and
+// Exec.
+type selectExecer interface {
```

Comment:
Any reason not to continue with the previous naming scheme here? The lack of the `db` prefix is a bit weird.